### DISCUSSION TITLE
How can I use URLs with .html extension in quasar app?

### DISCUSSION THREAD

**Discussion by iliubinskii** [quasarframework/quasar#12665] [CLI - PWA mode]
I am using quasar dev server.

This is what I get for URL without extension:
![image](https://user-images.githubusercontent.com/70687432/156872213-4509c57a-e672-42ef-aea8-7da41bd8fe85.png)

And this is for URL with extension:
![image](https://user-images.githubusercontent.com/70687432/156872236-fc7270e7-aa52-4e07-9052-fb8222c416ea.png)

this is my public/.htaccess file:
```
<IfModule mod_rewrite.c>
  <IfModule mod_negotiation.c>
    Options -Indexes -MultiViews
  </IfModule>

  RewriteEngine On

  RewriteCond %{REQUEST_FILENAME} !-d
  RewriteCond %{REQUEST_FILENAME} !-f
  RewriteRule ^ index.html [L]
</IfModule>
```

The question is why URLs with .html do not work.

**Comment by MilosPaunovic** [comment]
If you're using `vue-router`, it doesn't handle external links.
But, without you sharing some of the code where you do navigation it will be hard for anyone to help.
